sdk/webpubsub/Azure.Messaging.WebPubSub.Client/src/WebPubSubClient.cs

```diff
@@ -84,7 +241,48 @@ public virtual Task StartAsync(CancellationToken cancellationToken = default)
         public virtual Task StopAsync()
```

Comment:
stop does not stop the receiving task? It means although I stop the client, I am still handling the incoming messages?